### ISSUE TITLE
Titanic Example Freezes on Load Data

### ISSUE THREAD

**Issue by awcchungster** [kubeflow-kale/kale#65]
> kubeflow       titanic-ml-o56ks-44tt9-3397020455                              0/1     Completed   0          23s
> kubeflow       titanic-ml-o56ks-44tt9-4161189240                              0/2     Pending     0          20s
> kubeflow       titanic-ml-o56ks-44tt9-687348209                               0/1     Completed   0          23s

When I run the titanic-ML example, it freezes on the loaddata stage.

I suspect the problem is that my kale environment doesn't allow me to connect my existing storage from my notebook. The toggle grayed out. Any ideas why?

After I try tailing the logs for the container, I get the option wait | main. I try tailing the log on both, and it's empty. Does anyone know what's going on?
```
$ kubectl logs --namespace kubeflow titanic-ml-o56ks-44tt9-4161189240
Error from server (BadRequest): a container name must be specified for pod titanic-ml-o56ks-44tt9-4161189240, choose one of: [wait main]
```

**Comment by elikatsis** [comment]
Hi @awcchungster,
thanks for trying out the example.

What environment are you running on?
What options do you choose for your pipeline? Do you specify volumes? Do you choose with auto-snapshot  switched on?

>I suspect the problem is that my kale environment doesn't allow me to connect my existing storage from my notebook. The toggle grayed out. Any ideas why?

Could it be related to #30 ? That is, are you trying to mount your notebook's volumes?

**Comment by awcchungster** [comment]
I running on Kubeflow 1.0rc4 on top of EKS. 

The example I am doing follows the google workshop, and I'll attach a screenshot here. Since I couldn't access the notebook data, I changed the file reading to another source:

```
path = "data/"

PREDICTION_LABEL = 'Survived'

url1 = 'https://raw.githubusercontent.com/kubeflow-kale/examples/master/titanic-ml-dataset/data/test.csv'
url2 = 'https://raw.githubusercontent.com/kubeflow-kale/examples/master/titanic-ml-dataset/data/train.csv'
print("getting files")
urllib.request.urlretrieve(url1, path + "test.csv")
urllib.request.urlretrieve(url2, path + "train.csv")
print("reading files")
test_df = pd.read_csv(path + "test.csv")
train_df = pd.read_csv(path + "train.csv")
```

![Screen Shot 2020-02-19 at 9 47 41 AM](https://user-images.githubusercontent.com/3067000/74860095-ee366580-52fc-11ea-8e10-886e159dc19f.png)

**Comment by Edwardsleo** [comment]
Is this issue resolved ...?
Same issue facing in GKE Kubeflow Version : 0.7.1
kubeflow    titanic-ml-ldq9s-rsjgz-1420965606   0/2   Pending 0 61m

**Comment by awcchungster** [comment]
I still have the issue.

**Comment by elikatsis** [comment]
This must be related to the marshaling PVC not being created.
Please take a look at #18 #42 #51 where this has come up.

TL;DR: Your cluster should support `RWX` PVCs, or you should change the `.py` file to make the marshal volume a `RWO` PVC.
